tests/pytorch/test_to_device.py

```diff
@@ -0,0 +1,14 @@
+import dgl
+import torch
+
+def test_to_device():
+    g = dgl.DGLGraph()
+    g.add_nodes(5, {'h': torch.ones((5,2))})
+    g.add_edges([0,1],[1,2], {'m' : torch.ones((2,2))})
```

Comment:
code formating issue, add space after comma

Reply:
Test code is fine. It is not as strict as the codes in main package.

Reply:
Fixed